modules/core/04-channel/keeper/events.go

```diff
@@ -127,11 +127,12 @@ func EmitChannelCloseConfirmEvent(ctx sdk.Context, portID string, channelID stri
 
 // EmitSendPacketEvent emits an event with packet data along with other packet information for relayer
 // to pick up and relay to other chain
+
 func EmitSendPacketEvent(ctx sdk.Context, packet exported.PacketI, channel types.Channel, timeoutHeight exported.Height) {
 	ctx.EventManager().EmitEvents(sdk.Events{
 		sdk.NewEvent(
 			types.EventTypeSendPacket,
-			sdk.NewAttribute(types.AttributeKeyData, string(packet.GetData())), // DEPRECATED
+			sdk.NewAttribute(types.AttributeKeyData, string(packet.GetData())), //nolint:staticcheck // DEPRECATED
```

Comment:
Should these deprecation notices not be above and something like:

```
// Deprecated: types.AttributeKeyData is deprecated in favour of types.AttributeKeyDataHex
```

Even still, do we need deprecation comments here if the attribute key itself is marked as deprecated in the `types` package?

Reply:
honestly do not know